samples/bluetooth/peripheral_ancs_client/README.rst

```diff
@@ -0,0 +1,284 @@
+#. Press Button 2 to retrieve the app attributes and observe that you receive the display name for the app identifier from the previous step. For example, requesting the app attributes for "com.apple.mobilecal" yields "Calendar".
+#. If the notification has a flag for a positive or negative action, perform the notification action with Button 3 or Button 4, respectively.
+
+Testing with nRF Connect for Desktop
+------------------------------------
+
+1. |connect_terminal_specific|
+#. Reset the kit.
+#. Start `nRF Connect for Desktop`_ and select the connected dongle that is used for communication.
+#. Go to the **Server setup** tab.
+   Click the dongle configuration and select **Load setup**.
+   Load the ``ANCS_central.ncs`` file that is located under :file:`samples/bluetooth/peripheral_ancs_client` in the |NCS| folder structure.
+#. Click **Apply to device**.
+#. Go to the **Connection Map** tab.
+   Click the dongle configuration and select **Security parameters**.
+   Check **Perform Bonding**, and click **Apply**.
+#. Connect to the device from nRF Connect. The device is advertising as "ANCS".
+#. Wait until the bond is established. Verify that the UART data is received as follows::
+
+      Connected xx:xx:xx:xx:xx:xx (random)
+      The discovery procedure succeeded
+      Security changed: xx:xx:xx:xx:xx:xx (random) level 2
+      Pairing completed: xx:xx:xx:xx:xx:xx (random), bonded: 1
+
+#. After bonding, verify in nRF Connect that the Client Characteristic Configuration descriptors (CCCD) for ANCS Notification Source (0x120D) and ANCS Data Source (0xC6E9) are set to ``01 00``.
+
+Send an iOS notification to the application
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following table shows the format of a notification that you can send to the example application:
+
+   +------------------+---------------+--------------------------+
+   | Field            | Example value | Interpretation           |
+   +==================+===============+==========================+
+   | Event ID         | 0             | Notification added       |
+   +------------------+---------------+--------------------------+
+   | Flags            | 18            | Positive/negative action |
+   +------------------+---------------+--------------------------+
+   | Category         | 06            | Email                    |
+   +------------------+---------------+--------------------------+
+   | Category count   | 02            |                          |
+   +------------------+---------------+--------------------------+
+   | Notification UID | 01 02 03 04   | 67305985 (0x4030201)     |
+   +------------------+---------------+--------------------------+
+
+1. In nRF Connect, set the value of ANCS Notification Source (characteristic 0x120D) to ``00 18 06 02 01 02 03 04`` and click Update.
```

Comment:
```suggestion
1. In nRF Connect, set the value of ANCS Notification Source (characteristic 0x120D) to ``00 18 06 02 01 02 03 04`` and click :guilabel:`Update`.
```

Reply:
Done.

Besides, ANCS Notification Source is modified to a GUI label 'Apple Notification Source'. It aligns with nRF Connect for Desktop.